### ISSUE TITLE
Documentation Missing

### ISSUE THREAD

**Issue by krispayne** [brysontyrrell/PatchServer#34]
https://patchserver.readthedocs.io/en/latest/

There's no documentation here anymore

**Comment by brysontyrrell** [comment]
@krispayne Documentation has moved into the repository.
https://github.com/brysontyrrell/PatchServer/blob/develop/README.md

**Comment by krispayne** [comment]
I see.

I migrated over to the docker method and was curious what permissions should be on the persistent storage path, but I don't see that in the documentation.

Specifically, I'm getting `(sqlite3.OperationalError) attempt to write a readonly database` so I assumed it was permissions, but the persistent storage is owned by root (0:0) so I figured the docker container would see it correctly, so maybe it's a sqlite issue and not docker persistent storage/permissions issue?

This is the same DB I was previously using as a standard wsgi install, so maybe that has something to do with it
